howto/deploy-k8s-prod/README.md

```diff
@@ -0,0 +1,187 @@
+# Guidelines for production-ready deployments on Kubernetes
+
+This section outlines recommendations and practices for deploying Dapr to a Kubernetes cluster in a production-ready configuration.
+
+## Contents
+
+- [Cluster capacity requirements](#cluster-capacity-requirements)
+- [Sidecar resource requirements](#sidecar-resource-requirements)
+- [Deploying Dapr with Helm](#deploying-dapr-with-helm)
+- [Upgrading Dapr with Helm](#upgrading-dapr-with-helm)
+- [Recommended security configuration](#recommended-security-configuration)
+- [Tracing and metrics configuration](#tracing-and-metrics-configuration)
+
+## Cluster Capacity Requirements
+
+For a production ready Kubernetes cluster deployment, it is recommended you run a cluster of 3 nodes to support a highly-available setup of the control plane.
+The Dapr control plane pods are designed to be very lightweight and require the following resources in a production-ready setup:
+
+| Deployment  | CPU | Memory
+| ------------| ---- | ------
+| Operator | Limit: 1, Request: 100m | Limit: 200Mi, Request: 20Mi
```

Comment:
Can you provide a link to the units here? Is the CPU 1 core? Is the request 100m (what is m?). I added links to the K8s docs for these

Reply:
Added.